FluentAvalonia.sln

```diff
@@ -3,9 +3,17 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 16
 VisualStudioVersion = 16.0.30114.105
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "FluentAvalonia", "FluentAvalonia\FluentAvalonia.csproj", "{E527B96A-67A6-41C1-92A8-E56E341BB947}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "FluentAvalonia", "FluentAvalonia\FluentAvalonia.csproj", "{E527B96A-67A6-41C1-92A8-E56E341BB947}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "FluentAvaloniaSamples", "FluentAvaloniaSamples\FluentAvaloniaSamples.csproj", "{EDF76EAC-2EA9-49D5-9C33-372BC76305F2}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "FluentAvaloniaSamples", "FluentAvaloniaSamples\FluentAvaloniaSamples.csproj", "{EDF76EAC-2EA9-49D5-9C33-372BC76305F2}"
```

Comment:
Visual Studio changed these guids above.
I think it is reasonable to leave it as is - https://github.com/dotnet/project-system/issues/3079